routing/turns.cpp

```diff
@@ -52,6 +52,20 @@ static_assert(g_turnNames.size() == static_cast<size_t>(TurnDirection::Count),
 namespace routing
 {
 // UniNodeId -------------------------------------------------------------------
+std::atomic<uint64_t> UniNodeId::m_nextFakeId(0);
+
+UniNodeId::UniNodeId(FeatureID const & featureId, uint32_t startSegId, uint32_t endSegId,
+                     bool forward)
+  : m_type(Type::Mwm)
+  , m_featureId(featureId)
+  , m_startSegId(startSegId)
+  , m_endSegId(endSegId)
+  , m_forward(forward)
+{
+  if (!m_featureId.IsValid())
+    m_nodeId = static_cast<NodeID>(m_nextFakeId++);
```

Comment:
Am I right that NodeID is unsigned? If so, looks like m_nextFakeId should be of type NodeID too.